edgelet/aziot-edged/Cargo.toml

```diff
@@ -4,6 +4,9 @@ version = "0.1.0"
 authors = ["Azure IoT Edge Devs"]
 publish = false
 edition = "2021"
+description = "The aziot-edged is the IoT edge daemon."
```

Comment:
This description should be consistent with the changes you made to your PR in the identity service repo, e.g.,
```
description = "aziot-edged is the main binary for the IoT Edge daemon."
```